regression/clickDrag.side

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
i'm unclear on what this file is for - but if this is a snapshot, snapshot testing is inherently brittle and shouldn't be used. We'd need a real unit test.